source/objects/ptoSimClass.m

```diff
@@ -94,16 +94,16 @@
             'offset',                           'NOT DEFINED',...           % [m] Offset length
             'rodLength',                        'NOT DEFINED')              % [m] Rod length
         %name (1,:) {mustBeText}                 = 'NOT DEFINED'             % Electric Block Name
-        rectifyingCheckValve (1,1) struct       = struct(...                % hydraulic Block properties
-            'Cd',                               'NOT DEFINED',...           % Discharge accumulator
-            'Amax',                             'NOT DEFINED',...           % Maximum opening area of the valve
-            'Amin',                             'NOT DEFINED',...           % Minimum opening area of the valve
-            'pMax',                             'NOT DEFINED',...           % Pressure at maximum opening
-            'pMin',                             'NOT DEFINED',...           % Cracking pressure
-            'rho',                              'NOT DEFINED',...           % Fluid density
-            'k1',                               'NOT DEFINED',...           % Valve coefficiente
-            'k2',                               'NOT DEFINED')              % Valve coefficient, it's a function of the other valve variables
-        simpleDirectDrivePTO (1,1) struct      = struct(...                 % simple direct drive linear PTO Block properties
+        rectifyingCheckValve (1,1) struct       = struct(...                % Rectifying Check Valve Block properties
+            'Cd',                               'NOT DEFINED',...           % [1] Discharge coefficient
+            'Amax',                             'NOT DEFINED',...           % [m^2] Maximum opening area of the valve
```

Comment:
@jleonqu this looks good, thank you. Can you clarify the `[1]` in front of `Discharge Coefficient` and `Valve Coefficient`?

Reply:
@kmruehl These are nondimensional values. I added a comment to all the nondimensional values for clarification.